cuqi/geometry/_geometry.py

```diff
@@ -437,11 +437,16 @@ class Image2D(Geometry):
         If order = 'C', the image is represented in row-major order.
         if order = 'F', the image is represented column-major order.
 
+    visual_only : bool, Default: False
```

Comment:
Do we need to add a check in CUQIarray when is_par is False 

```python
CUQIarray(np.ones(N,N), is_par=False, geom = geom)
```

where geom is an Image geometry for example.

Reply:
Good point. I realized that we need to switch fun_shape depending on only_visual. Its now updated.